### ISSUE TITLE
Kobolds with significant posing/world interactions explode on save/load

### ISSUE THREAD

**Issue by Luingar** [naelstrof/KoboldKare#55]
If you pose a kobold sticking through the fence and anchor many of its body parts (Head, chest, upper and lower legs, ass, tail, arms, and more), when you save and then load the game, the kobold will make an elastic snapping sound and be gone. I'm not sure if it's getting deleted, or launched somewhere, or what, but it's very much instantly gone.

Not sure if the issue is world geometry interaction, or just all the anchor points, or something else entirely.

**Comment by Godeken** [comment]
I had this issue too... pinned all my males against the wall of the starter house to use as a reference to see how many different types of "shaft" i can find... then saved and closed... opened up again an hour later and my character and others that were spread throughout the farm i was covered in explosion residue and all 6 pinned Kobolds were gone.

**Comment by Godeken** [comment]
Just tested it now with the latest build from the GIT in Unity... if I pose a Kobold in a custom pose, save and then load that save... the Kobold is gone... It didnt cause the explosions though...

**Comment by naelstrof** [comment]
Yeah-- posing isn't saved right now as far as I know. Ragdolling is though, and ragdolling does try to sample the character's position before it triggers. There's probably a race condition with loading the player's position *after* the ragdoll has already triggered. I think I know how to fix it.
